site/styles/_meta-detail.scss

```
@@ -590,3 +576,15 @@
     font-size: 12px;
   }
 }
+
+s72-element-switcher {
+  @extend .mb-5;
+}
+
+.meta-detail-infos {
+  @include media-breakpoint-up(lg) {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 15px;
+  }
+}
```

Probably nicer to consistently use grid across breakpoints:
```suggestion
.meta-detail-infos {
  display: grid;
  grid-template-columns: repeat(1, 1fr);
  gap: 20px;

  @include media-breakpoint-up(lg) {
    grid-template-columns: repeat(2, 1fr);
  }
}
```

---

Also... if you change the `gap` to `20px` and remove `padding-bottom` from line 279, it matches the designs a bit better.

---

- suggestion committed and padding-bottom removed